packages/theme/template/package.json

```diff
@@ -38,10 +38,10 @@
     "react": "^18.2.0"
   },
   "peerDependencies": {
-    "ilib": "^14.14.0 || ^14.14.0-webos1"
+    "ilib": "^14.15.1 || ^14.15.1-webos1"
```

Comment:
latest version of ilib-webos is 14.15.1 https://www.npmjs.com/package/ilib-webos?activeTab=versions

Reply:
Good, we need to check this.